docs/source/usage/security_recommendations.rst

```diff
@@ -112,5 +112,177 @@ Note:
     * factory_reset's priority shouldn't be higher than deploy.erase_devices_metadata's priority, or it may cause issues during erasing metadata. If the bios interface is set to no-bios, the factory_reset cleaning step won't be executed.
     * Applying specific BIOS settings to nodes is not supported by automated cleaning. Owner should do it manually.
 
+iDRAC Settings Reset
+--------------------
+
+A lessee leasing a node can change the iDRAC settings of the leased node. If these settings are not reset after the node is returned, then this can result in unexpected behavior for future users of the node. A node Owner or Admin can back up the iDRAC settings before leasing a node and restore the settings after the lease ends. Two ways to import/export iDRAC settings:
+
+* Redfish API
+* OpenStack Ironic Bare Metal Cleaning service.
+
+
+Prerequisites
+~~~~~~~~~~~~~
+
+The `iDRAC`_ hardware type requires the python-dracclient and the sushy library to be installed on the Ironic conductor node(s) if the node is configured to use an idrac-redfish or idrac-wsman interface implementation.
+
+* Install the `required libraries`_ and `enable iDRAC hardware type`_ with the required interfaces.
+
+* Restart the Ironic conductor service for the new configuration to take effect.
+
+Configure the node appropriately:
+ The following command configures a bare metal node with the idrac hardware type assuming a mix of Redfish and WSMAN interfaces are used:
+
+.. prompt:: bash $
+
+  openstack baremetal node set \
+    --driver idrac \
+    --driver-info drac_username=user \
+    --driver-info drac_password=pa$$w0rd
+    --driver-info drac_address=drac.host \
+    --driver-info redfish_username=user \
+    --driver-info redfish_password=pa$$w0rd \
+    --driver-info redfish_address=drac.host \
+    --driver-info redfish_system_id=/redfish/v1/Systems/System.Embedded.1 \
+    --bios-interface idrac-redfish \
+    --inspect-interface idrac-redfish \
+    --management-interface idrac-redfish \
+    --power-interface idrac-redfish
+    <node>
+
+
+Import or export iDRAC configuration with idrac-redfish:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sample scripts to export and import the iDRAC configuration using the `Redfish API`_ are shown below. For more information refer the Redfish Module `function definitions`_.
+
+* Export configuration:
+
+.. prompt:: bash $
+
+  python3 ExportSystemConfigurationLocalREDFISH.py -ip '10.1.10.3' -u 'new-user' -p $ADMIN_PASSWORD -t IDRAC
+
+
+* Import configuration after making changes:
```

Comment:
The "making changes" here should maybe be its own bullet point before this; after you export the configuration, you make changes, and then you import it. Under the "Make changes" bullet point you can talk a little about reasons why; getting rid of node-specific items or passwords, etc.

Reply:
Done